test/integration/orchestrator-spec.ts

```diff
@@ -1,34 +1,47 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
were there a lot of these violations in this file?

Reply:
Yeah the way `MockContext` is written it has a lot of `null` properties that are used by `!.` which is a bit of a no-no (you should test for null first because otherwise there's little point in it being nullable)